dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/TeradataUtils.java

```diff
@@ -30,6 +30,10 @@ public static <T> Optional<T> optionalIf(boolean condition, Supplier<T> supplier
     return condition ? Optional.of(supplier.get()) : Optional.empty();
   }
 
+  public static Optional<String> nonEmpty(String value) {
+    return value != null && !value.isEmpty() ? Optional.of(value) : Optional.empty();
```

Comment:
optional: You could use: `Optional.ofNullable(Strings.emptyToNull(value))`

Reply:
Done.